### DISCUSSION TITLE
Just upgraded from Mint 21.3 to Mint 22. Conky spacing now off and cpugraph permanently red?

### DISCUSSION THREAD

**Discussion by dagleykd** [brndnmtthws/conky#2067] [Q&A] [answered]
Hi all - title says it all. I just upgraded from Mint 21.3 to Mint 22 today on my HTPC. All went mostly well but I had to reinstall Conky and Conky Manager2. I have them back and loaded my usual setup, but the spacing was all off....managed to fix that with playing around, but I CANNOT, no matter what I do, get my cpugraph that shows the Core 0 cpu usage to display in a gradient white/gray like I want. Since the upgrade it only displays in red.

I know the conky version I had in Mint 21.3 was conky 1.12, so old. The Conky version I have now is Conky 1.19.

I had found some posts talking about the same issue I am having and telling people to specify hex colors for the graph, but as far as I know I am now doing that and is still showing red and not the colors I have set. It is also ignoring the current height I have set?

This is my line of code that specifically mentions the graph:
`Activity: ${voffset -16}${cpugraph 0 10,137 #666666 #FFFFFF -l}`

My other graphs for upload speed and download speed are working perfectly and are formatted the same way in terms of color. Am I overthinking this or missing something?

My theme is a heavily personalized version of the old Minimalis theme.

Any help is appreciated! :)

**Comment by Caellian** [comment, marked as answer]
I'd say you're missing something - [updates](https://github.com/brndnmtthws/conky/compare/v1.19.0...v1.21.7). The version you have is over a year old at this point, the cpugraph was changed a lot, the spacing issues (definitely a bug, introduced by me :man_cartwheeling: ) were fixed, as well as a whole bunch of other issues.

My suggestion is you [get `deb-get`](https://forums.linuxmint.com/viewtopic.php?t=378943) and update conky (`conky-all` package) from Debian repository which has [1.21.6-1](https://packages.debian.org/trixie/any/conky-all). Alternatively download the latest one ([1.21.7 gzip link](https://github.com/brndnmtthws/conky/releases/download/v1.21.7/conky.1.gz)) and just overwrite the system installation. `deb-get` is probably a better option because it will make you package manager aware of the update, so I recommend you do that, but overwriting system install will also work because it's a leaf dependency and nothing depends on it having stable ABI.

Once you've done that, and reverted fixes you did to get the bad spacing code working (sorry for that), you should be good. Lmk if cpugraph issue persists and I'll open an issue for you.
